File: java/program59.java
import java.util.*;

class StringX
{
    public void CountCapitalSmall(String str)
    {
        char Arr[] = str.toCharArray();
        int small = 0;
        int capital = 0;

        for(int i = 0; i < Arr.length; i++)
        {
            if((Arr[i] >= 'A') && (Arr[i] <= 'Z'))   
            {
                capital++;
            }
            else if((Arr[i] >= 'a') && (Arr[i] <= 'z'))
            {
                small++;
            }
        }
        System.out.println("count of the small letters are : "+small);
        System.out.println("count of the capital letters are : "+capital);
    }
}

class program59
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String ");
        String str = sobj.nextLine();
        
        StringX obj = new StringX();
        obj.CountCapitalSmall(str);
        
    }
}

File: cpp/program59.cpp
#include<iostream>
using namespace std;

class StringX
{
    public:

        void CountCapitalSmall(char *str)
        {
            int i = 0 , j = 0;

            while(*str != '\0')
            {
                if((*str >= 'A') && (*str <= 'Z'))
                {
                    i++;
                }
                else if((*str >= 'a') && (*str <= 'z'))
                {
                    j++;
                }
                str++;
            }
            cout<<"Count of the Capital letters is : "<<i<<"\n";
            cout<<"Count of the Small letters is : "<<j<<"\n";
        }

};

int main()
{
    char Arr[20];

    cout<<"Enter string\n";
    cin.getline(Arr,20);

    StringX obj;
    obj.CountCapitalSmall(Arr);
    
    return 0;
}